### ISSUE TITLE
Allow also classic USB connections while ippusbxd is running

### ISSUE THREAD

**Issue by tillkamppeter** [OpenPrinting/ippusbxd#9]
In some cases, for example on a multi-function device which also scans (at least as long the devices do not support driverless IPP scanning), it can make sense to allow classic USB connections while ippusbxd is running.
To do so, one could close the USB connection but keep the ippusbxd daemon running and listening when all conversations (threads) have closed, so that while not printing conventional USB is possible. Alternatively one could try to use only 2 of the 3 channels to keep one open for conventional USB, or use even both techniques to always use the most appropriate for the situation (for example also printing and scanning in parallel).
@DavieV, could you have a look into this?

**Comment by DavieV** [comment]
Yes I think this is doable, I'll look into implementing it when I have the time.

**Comment by tillkamppeter** [comment]
@DavieV, this issue is solved by @alexpevzner's [ipp-usb](https://github.com/alexpevzner/ipp-usb). I have tested it and while the ipp-usb daemon is running I can discover the printer and even print with the CUPS "usb" backend. I could even print with the "usb" backend during a running scan via SANE and ipp-usb.
